### PR TITLE
Rendering Elements

### PR DESCRIPTION
**Pull request by tamzidkarim**

### CODE REVIEW

File: content/docs/rendering-elements.md [hunk 1]
```diff
@@ -1,75 +1,76 @@
...
   - "docs/displaying-data.html"
 prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
+Element হল React এপ্লিকেশনের সবথেকে ছোট বিল্ডিং ব্লক।
 
-An element describes what you want to see on the screen:
+একটি element হল আপনি স্ক্রিনে কী দেখতে চান তা বর্ণ্না করেঃ
```
Comment (shakib609) [inline content/docs/rendering-elements.md:13]:
```suggestion
একটি element আপনি স্ক্রিনে কী দেখতে চান তা বর্ণনা করেঃ
```

File: content/docs/rendering-elements.md [hunk 2]
```diff
@@ -1,75 +1,76 @@
...
 
 ```js
-const element = <h1>Hello, world</h1>;
+const element = <h1>হ্যালো, ওয়ার্ল্ড</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+React element গুলো সাধারণ অবজেক্ট, এবং তৈরি করেতে সহজ, যা ব্রাউজারের DOM element এর মত নাহ। React element এর সাথে সমকক্ষতা রেখে DOM আপডেট করে React DOM।
```
Comment (shakib609) [inline content/docs/rendering-elements.md:19]: নাহ/না?
What do you suggest @nutboltu vai

File: content/docs/rendering-elements.md [hunk 3]
```diff
@@ -1,75 +1,76 @@
...
 
->**Note:**
+>**বিঃদ্রঃ**
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+>কেউ হয়ত বহুল পরিচিত "কম্পোনেন্ট" এর সাথে element গুলিয়ে ফেলতে পারে। [পরবর্তী বিভাগ](/docs/components-and-props.html) এ আমরা কম্পোনেন্ট পরিচয় করিয়ে দেব । কম্পোনেন্ট "তৈরি" হয় element দিয়ে, এবং এগিয়ে যাওয়ার আগে আমরা আপনাকে এই বিভাগটি পড়তে উত্সাহিত করি.
```
Comment (shakib609) [inline content/docs/rendering-elements.md:23]:
```suggestion
>কেউ হয়ত বহুল পরিচিত "কম্পোনেন্ট" এর সাথে element গুলিয়ে ফেলতে পারে। [পরবর্তী অনুচ্ছেদ](/docs/components-and-props.html) এ আমরা কম্পোনেন্ট পরিচয় করিয়ে দেব । কম্পোনেন্ট "তৈরি" হয় element দিয়ে, এবং এগিয়ে যাওয়ার আগে আমরা আপনাকে এই বিভাগটি পড়তে উৎসাহিত করি।
```

File: content/docs/rendering-elements.md [hunk 4]
```diff
@@ -1,75 +1,76 @@
...
 [](codepen://rendering-elements/render-an-element)
 
-It displays "Hello, world" on the page.
+এটি পেজ এ "Hello, world" প্রদর্শন করে।
 
-## Updating the Rendered Element {#updating-the-rendered-element}
+## রেন্ডারকৃত element আপডেট করা {#updating-the-rendered-element}
 
-React elements are [immutable](https://en.wikipedia.org/wiki/Immutable_object). Once you create an element, you can't change its children or attributes. An element is like a single frame in a movie: it represents the UI at a certain point in time.
+React element গুলা [immutable](https://en.wikipedia.org/wiki/Immutable_object)। একবার element বানান হয়ে গেলে, আপনি এর children অথবা attributes পরিবর্ত্ন করতে পারবেন না. একটি element হল সিনেমার একটি একক ফ্রেম এর মতঃ এটি UI এর একটি নির্দিষ্ট সময়কে উপস্থাপিত করে।
```
Comment (shakib609) [inline content/docs/rendering-elements.md:48]:
```suggestion
React element গুলা [immutable](https://en.wikipedia.org/wiki/Immutable_object)। একবার element বানানো হয়ে গেলে, আপনি এর children অথবা attributes পরিবর্তন করতে পারবেন না। একটি element হল সিনেমার একটি একক ফ্রেমের মতঃ এটি UI এর একটি নির্দিষ্ট সময়কে উপস্থাপিত করে।
```

File: content/docs/rendering-elements.md [hunk 5]
```diff
@@ -1,75 +1,76 @@
...
 [](codepen://rendering-elements/update-rendered-element)
 
-It calls [`ReactDOM.render()`](/docs/react-dom.html#render) every second from a [`setInterval()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval) callback.
+এটি  প্রতি সেকেন্ডে [`setInterval()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval) কলব্যাক থেকে [`ReactDOM.render()`](/docs/react-dom.html#render) কল করে।
 
->**Note:**
+>**বিঃদ্রঃ**
 >
->In practice, most React apps only call [`ReactDOM.render()`](/docs/react-dom.html#render) once. In the next sections we will learn how such code gets encapsulated into [stateful components](/docs/state-and-lifecycle.html).
+>চর্চার খাতিরে, অধিকাংশ React এপ্লিকেশন শুধু একবার [`ReactDOM.render()`](/docs/react-dom.html#render) কল করে। পরবর্তী বিভাগে আমরা শিখব এই জাতীয় কোড কীভাবে  [স্টেটফুল কম্পোনেন্ট](/docs/state-and-lifecycle.html) এ encapsulated হয়।
```
Comment (shakib609) [inline content/docs/rendering-elements.md:62]:
```suggestion
>চর্চার খাতিরে, অধিকাংশ React এপ্লিকেশন শুধু একবার [`ReactDOM.render()`](/docs/react-dom.html#render) কল করে। পরবর্তী বিভাগে আমরা শিখব এই জাতীয় কোড কীভাবে  [stateful কম্পোনেন্ট](/docs/state-and-lifecycle.html) এ encapsulated হয়।
```

File: content/docs/rendering-elements.md [hunk 6]
```diff
@@ -1,75 +1,76 @@
...
 >
->We recommend that you don't skip topics because they build on each other.
+>আমরা আপনাকে পরামর্শ দিই যে আপনি বিষয়গুলি এড়িয়ে যাবেন না কারণ তারা একে অপরকে তৈরি করে।
```
Comment (shakib609) [inline content/docs/rendering-elements.md:64]:
```suggestion
>আমরা আপনাকে পরামর্শ দিই যে আপনি বিষয়গুলি এড়িয়ে যাবেন না কারণ তারা একে অপরের উপর নির্ভরশীল।
```

Comment (shakib609) [reply content/docs/rendering-elements.md:64]: Also, there's an extra line in your PR:
2020-05-22-004503_106x23_scrot

File: content/docs/rendering-elements.md (hunk 2 above)
Comment (nutboltu) [reply content/docs/rendering-elements.md]: I would suggest `নয়`

File: content/docs/rendering-elements.md [hunk 7]
```diff
@@ -1,75 +1,76 @@
...
-An element describes what you want to see on the screen:
+একটি element আপনি স্ক্রিনে কী দেখতে চান তা বর্ণনা করেঃ
 
 ```js
-const element = <h1>Hello, world</h1>;
+const element = <h1>হ্যালো, ওয়ার্ল্ড</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+React element গুলো সাধারণ অবজেক্ট, এবং তৈরি করেতে সহজ, যা ব্রাউজারের DOM element এর মত না। React element এর সাথে সমকক্ষতা রেখে DOM আপডেট করে React DOM।
```
Comment (shakib609) [inline content/docs/rendering-elements.md:19]:
```suggestion
React element গুলো সাধারণ অবজেক্ট, এবং তৈরি করতে সহজ, যা ব্রাউজারের DOM element এর মত নয়। React element এর সাথে সমকক্ষতা রেখে DOM আপডেট করে React DOM।
```

Comment (shakib609) [reply content/docs/rendering-elements.md:19]: Also, the extra new-line needs to be removed.
2020-05-22-004503_106x23_scrot
Both numbers should be equal.

Comment (tamzidkarim) [reply content/docs/rendering-elements.md]: done.

File: content/docs/rendering-elements.md [hunk 8]
```diff
@@ -34,8 +34,7 @@ React element গুলো সাধারণ অবজেক্ট, এবং 
 
 শুধু React দ্বারা নির্মিত এপ্লিকেশন গুলোতে সাধারণত একটিমাত্র রুট DOM নোড থাকে। আপনি যদি কোন বিদ্যমান এপ্লিকেশনে React একীভূত করেন, তাহলে আপনি যত গুলা খুশী ভিন্ন রুট DOM নোড রাখতে পারেন।
 
-একটি React element রুট DOM নোড এ রেন্ডার করতে, উভয়কে [`ReactDOM.render()`](/docs/react-dom.html#render) এ pass করতে
-হবে:
+একটি React element রুট DOM নোড এ রেন্ডার করতে, উভয়কে [`ReactDOM.render()`](/docs/react-dom.html#render) এ pass করতে হবে:
```
Comment (shakib609) [inline content/docs/rendering-elements.md:37]: Another minor edit.
```suggestion
একটি React element রুট DOM নোড এ রেন্ডার করতে, উভয়কে [`ReactDOM.render()`](/docs/react-dom.html#render) এ pass করতে হবেঃ
```

**Comment by shakib609** [comment]
@nutboltu vai can you also do a quick check? In case, I missed something.

File: content/community/examples.md [hunk 9]
```diff
@@ -10,7 +10,7 @@ There are many example projects created by the React community. We're keeping th
 
 If you add a project, please commit to keeping it up to date with the latest versions of React.
 
-## Small Examples
+## Small Examples {#small-examples}
```
Comment (nutboltu) [inline content/community/examples.md:13]: @tamzidkarim any reason behind these changes in this file?

File: content/docs/rendering-elements.md [hunk 10]
```diff
@@ -1,75 +1,75 @@
...
 
-Elements are the smallest building blocks of React apps.
+Element হল React এপ্লিকেশনের সবথেকে ছোট বিল্ডিং ব্লক।
 
-An element describes what you want to see on the screen:
+একটি element আপনি স্ক্রিনে কী দেখতে চান তা বর্ণনা করেঃ
 
 ```js
-const element = <h1>Hello, world</h1>;
+const element = <h1>হ্যালো, ওয়ার্ল্ড</h1>;
```
Comment (nutboltu) [inline content/docs/rendering-elements.md:16]: Sorry that was my mistake that I did in the `Home Page`. But let's not translate code snippet.

Comment (tamzidkarim) [reply content/docs/rendering-elements.md:16]: No problem.

File: content/community/examples.md (hunk 9 above)
Comment (tamzidkarim) [reply content/community/examples.md:13]: Honestly I dont have any idea how this file got changed. So thats why i kept it that way. Now that you mentioned it I guess I will unchange this file.

**Pull request merged**